storybook/stories/components/AppBarInternal/AppBarInternal.js

```diff
@@ -0,0 +1,102 @@
+import { LogoJPLTemplate } from '../LogoJPL/LogoJPL'
+import { IconCloseTemplate } from '../Icons/IconClose'
+import { IconMenuTemplate } from '../Icons/IconMenu'
+import { IconUserTemplate } from '../Icons/IconUser'
+
+export const AppBarInternalTemplate = ({
+  orgNumber,
+  siteTitle,
+  includeMobileMenuToggle,
+  includeSignIn,
+  signedIn,
+  userData,
+  logInUrl,
+  logOutUrl,
+}) => {
+  // prop defaults
+  if (!orgNumber) orgNumber = ''
+  if (!siteTitle) siteTitle = ''
+  if (includeMobileMenuToggle == undefined) includeMobileMenuToggle = false
+  if (includeSignIn == undefined) includeSignIn = false
+  if (signedIn == undefined) signedIn = false
+  if (!userData) userData = {}
+  if (!logInUrl) logInUrl = '#'
+  if (!logOutUrl) logOutUrl = '#'
+
+  const mobileMenuToggleTemplate = includeMobileMenuToggle
+    ? `<button id="NavMobileToggle" class="block lg:hidden focus:outline-none focus:ring-2 p-2" aria-label="Open menu">
+      ${IconMenuTemplate({})}
+      ${IconCloseTemplate({})}
+    </button>
+    `
+    : ``
+
+  const signInTemplate = includeSignIn
+    ? signedIn && userData
+      ? `
+        <a href="${logOutUrl}" class="flex flex-nowrap items-center focus:outline-none focus:ring-2 mr-2 py-1">
+          <span class="hidden lg:block mr-2">
+            Sign Out
+          </span>
+          ${
+            userData.photo
+              ? `<span class="block flex-shrink-0 w-6 h-6 overflow-hidden rounded-full border-2 border-white bg-white">
+              <img class="object-cover w-6 h-6" src="${userData.photo}" alt="${userData.firstName} ${userData.lastName}'s profile photo" />
+            </span>`
+              : `
+            <span class="block flex-shrink-0">
+              ${IconUserTemplate({})}
+            </span>
+            `
+          }
+
+        </a>
+      `
+      : `
+      <a href="${logInUrl}" class="flex flex-nowrap items-center focus:outline-none focus:ring-2 mr-2 py-1">
+        <span class="hidden lg:block mr-2">
+          Sign In
+        </span>
+        <span class="block flex-shrink-0">
+          ${IconUserTemplate({})}
+        </span>
+      </a>
+    `
+    : ``
+
+  return `
+  <div class="AppBarInternal sticky lg:relative top-0 z-30">
+    <div class="bg-gradient-to-r from-jpl-red to-jpl-red-darker text-white font-medium relative z-10">
+      <div class="flex flex-row px-4 lg:container mx-auto py-2 items-center justify-between">
+        <div class="flex flex-row items-center py-px">
+          <a href="//js.jpl.nasa.gov" aria-label="JPL Space" class="block lg:text-sm mr-10">
```

Comment:
Is there a reason to set this text size at all anymore?

```suggestion
          <a href="//js.jpl.nasa.gov" aria-label="JPL Space" class="block mr-10">
```

Reply:
Good catch!